src/components/command-bar/commands/search/helpers/generate-suggested-pages.tsx

```diff
@@ -0,0 +1,257 @@
+import { IconDocs16 } from '@hashicorp/flight-icons/svg-react/docs-16'
+import { IconDownload16 } from '@hashicorp/flight-icons/svg-react/download-16'
+import { IconGuide16 } from '@hashicorp/flight-icons/svg-react/guide-16'
+import { IconLearn16 } from '@hashicorp/flight-icons/svg-react/learn-16'
+import { ProductSlug } from 'types/products'
+import { productSlugsToNames } from 'lib/products'
+import ProductIcon from 'components/product-icon'
+import { SuggestedPage } from '../suggested-pages'
+
+const generateTutorialLibrarySuggestedPage = (productSlug?: ProductSlug) => {
+	// TODO abstract this, or leverage abstraction from tutorial library feature
+	let url = '/tutorials/library'
+	if (productSlug) {
+		url += `?product=${productSlug}`
+	}
+
+	return {
+		icon: <IconGuide16 />,
+		text: 'Tutorial Library',
+		url,
+	}
+}
+
+const DEFAULT_SUGGESTED_PAGES = [
+	...[
+		'hcp',
+		'terraform',
+		'packer',
+		'vault',
+		'boundary',
+		'consul',
+		'nomad',
+		'waypoint',
+		'vagrant',
+	].map((productSlug: ProductSlug) => ({
+		icon: <ProductIcon productSlug={productSlug} />,
+		text: productSlugsToNames[productSlug],
+		url: `/${productSlug}`,
+	})),
+	generateTutorialLibrarySuggestedPage(),
+]
+
+const generateBasicSuggestedPages = (productSlug: ProductSlug) => {
+	const pages = [
+		{ icon: <IconDocs16 />, text: 'Documentation', path: 'docs' },
+		{ icon: <IconDownload16 />, text: 'Install', path: 'downloads' },
+		{ icon: <IconLearn16 />, text: 'Tutorials', path: 'tutorials' },
+	].map(({ icon, text, path }) => {
+		let fullText
+		if (productSlug === 'hcp') {
+			fullText = `HCP ${text}`
+		} else {
+			fullText = `${productSlugsToNames[productSlug]} ${text}`
+		}
+
+		return {
+			icon,
+			text: fullText,
+			url: `/${productSlug}/${path}`,
+		}
+	})
+
+	if (productSlug === 'hcp') {
+		return pages.filter((page) => page.url !== '/hcp/downloads')
+	}
+
+	return pages
+}
+
+/**
+ * These are pages listed after the main pages for a product, and just before a
+ * link to the Ttorials Library.
+ */
+const EXTRA_PAGES: Record<Exclude<ProductSlug, 'sentinel'>, SuggestedPage[]> = {
```

Comment:
🗒️ Ideally these will live in `src/content` in the future, but I don't want to add something there that could cause any confusion while content work is ongoing.